server/main.py

```diff
@@ -94,3 +121,28 @@ async def calculate_spectrum(payload: Payload):
                 "units": spectrum.units[payload.mode],
             },
         }
+
+#[Date] _[database] _[molecule]_[temperature]K_[pressure]atm
+@app.post("/download-spectrum")
+async def download_spec(payload: Payload):
+    print(payload)
+    date = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S");
+    file_name = f"downloaded_spectrum/{date}_{payload.database}_{payload.tgas}k_{payload.pressure}atm.spec"
```

Comment:
Does s.get_name() not work? What does it output for this?

Reply:
you are sure that it is going to create unique file names